packages/client/src/app/challenge/components/ChallengeCheckBox.vue

```diff
@@ -64,13 +64,18 @@ export default {
     }
   },
   computed: {
-    getClass() {
+    getButtonClass() {
       return this.checked
         ? 'position-absolute top-50 text-tertiary'
```

Comment:
You can also choose to only return the class that changes and put the other classes in the template, if they won't charge depending on the checked start state

Reply:
Good point, adjusted